### PR TITLE
Feature/add doc container

### LINKED ISSUE #19: Add documentation Container
Add documentation Container

### PR DESCRIPTION
**Pull request by claudioed**
Added a docs container to users to be able to see the documentation in a more friendly way

### CODE REVIEW

File: pkg/controller/apimock/pod.go [hunk 1]
```diff
@@ -21,76 +21,123 @@ import (
...
 		Spec: v1.PodSpec{
-			Containers: []v1.Container{mockContainer(mock)},
+			Containers: []v1.Container{mockContainer(mock), docContainer(mock)},
 			Volumes:    volumes,
 		},
 	}
 }
 
+// create mock container, it will deploy the mock api
 func mockContainer(mock *v1alpha1.APIMock) v1.Container {
```
Comment (angelokurtis) [inline pkg/controller/apimock/pod.go]: Unused parameter

File: pkg/controller/apimock/pod.go [hunk 2]
```diff
@@ -21,76 +21,123 @@ import (
...
+		},
+		VolumeMounts: volumeMount(),
+		Ports:        ports,
+		Resources:    requirements(),
+		Env:          []v1.EnvVar{cnPort},
+	}
+}
+
+// create documentation container, it will used for display the swagger-ui
+func docContainer(mock *v1alpha1.APIMock) v1.Container {
```
Comment (angelokurtis) [inline pkg/controller/apimock/pod.go]: Unused parameter

File: pkg/controller/apimock/pod.go (hunk 1 above)
Comment (claudioed) [reply pkg/controller/apimock/pod.go]: Done!

File: pkg/controller/apimock/pod.go (hunk 2 above)
Comment (claudioed) [reply pkg/controller/apimock/pod.go]: Done!

**Pull request merged**